index.js

```diff
@@ -0,0 +1,55 @@
+const form = document.querySelector('#form');
+const titleInput = document.querySelector('#title');
+const authorInput = document.querySelector('#author');
+const booksContainer = document.querySelector('.books-container');
+
+// collection that keeps a list of books
+let collection = JSON.parse(localStorage.getItem('bookList')) || [];
+
+// create the book element
+const createBookelement = ({ title, author }) => {
+  const booksList = document.createElement('div');
+  booksList.innerHTML = `<h3 class="title">${title}</h3>
+    <p class="author">${author}</p>
+    <button class="delete" type="button">Remove</button><br><br><hr>`;
+  booksContainer.appendChild(booksList);
+
+  document.querySelectorAll('.delete').forEach((btn) => {
+    btn.addEventListener('click', (e) => {
+      const targetTitle = e.target.parentElement.firstChild.textContent;
+      const targetAuthor = e.target.previousElementSibling.textContent;
+      const targetEl = e.target.parentElement;
+      collection = collection.filter(
+        (book) => book.title !== targetTitle || book.author !== targetAuthor,
+      );
+      localStorage.setItem('bookList', JSON.stringify(collection));
+
+      targetEl.remove();
+    });
+  });
```

Comment:
- [ ] **[OPTIONAL]** Take a look at the image below. I added duplicate books, but when I delete one of them all books in the storage with the same title and author are cleared. This is because your remove function is filtering books with the same title or author. I would recommend you attach some kind of unique `id` to your books on creation and implement your deletion using the `id` instead of title or author.

  ![denscholar-book](https://user-images.githubusercontent.com/36542986/148867626-a2b32c57-e984-405c-b80f-8aa9720cec8c.jpg)

Reply:
@TedLivist Noted with thanks. I will research about this.